source/activity/view-logs.txt

```diff
@@ -1,26 +1,141 @@
-==============================
-Access Logs with the Admin API
-==============================
+.. _view-logs:
 
-.. default-domain:: mongodb
+=====================
+View Application Logs
+=====================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 2
    :class: singlecol
 
-Overview
---------
+App Services UI
+---------------
 
-You can request an application's logs programmatically by accessing the
-logging endpoints of the :doc:`Atlas App Services Admin API </admin/api/v3>`.
+You can view, filter, search, and download an application's logs in the
+App Services UI. To navigate to the logs screen, click :guilabel:`Logs`
```

Comment:
"To navigate to the logs screen..." we don't format or capitalize screen names at all? Just `the logs screen` reads oddly to me.

Reply:
Yeah - rephrasing to be about "logs" and not "the Logs screen"